### ISSUE TITLE
All plan action should be reversible until the plan is locked in

### ISSUE THREAD

**Issue by frnsys** [frnsys/half_earth#43]
- [x] You should be able to re-convert research/initiative points back into PC, but only for "new" points, i.e. points you've converted in the current planning session.
- [x] You should be able to "undo" policies implemented in the current planning session (i.e. get a refund)
- [x] You should be able to "undo" upgrades made in this session (i.e. get a refund)
- [x] Withdrawn policies can be re-implemented free of charge
- [x] You can undo downgrades made in this session (i.e. reimplement the upgrades free of charge)

Basically, plan decisions are only final once the player hits the "Ready" button

Process changes are already "reversible" since they don't cost any PC

**Comment by frnsys** [comment]
One difficult thing here is how to communicate changes if the plan were to be implemented? Right now you have instant feedback--if you implement something that e.g. changes contentedness, you see the impact right away. But if we had this "shopping cart" approach instead, we'd need to communicate the potential change of the current changes in your "cart".

This might be easier with the design that Son La has in the works, where you "swipe"/"punch" the card to add it to the plan. Possibly when you swipe it 50% of the way we give a preview of its effects before you fully lock it in.
